docs/tutorials/infer_phys_pars.rst

```diff
@@ -0,0 +1,503 @@
+
+The position angle of the screen
+--------------------------------
+
+The first physical parameter to infer from the free parameters of our model is
+the position angle of the screen :math:`\xi`. This parameter can be computed
+from the fitted phase offset of Earth's orbital velocity signature
+:math:`\chi_\mathrm{E}` and the known orientation of Earth's orbit
+(:math:`i_\mathrm{E}` and :math:`\Omega_\mathrm{E}`), using the equation
+
+.. math::
+
+    \xi = \Omega_\mathrm{E} + \Delta\Omega_\mathrm{E},
+    \qquad \mathrm{with} \qquad
+    \tan( \Delta\Omega_\mathrm{E} ) = \frac{ \tan( \chi_\mathrm{E} ) }
+                                           { \cos( i_\mathrm{E} ) }.
+
+Note that for a given value of :math:`\chi_\mathrm{E}`, there are two possible
+solutions to the right-hand-side equation for :math:`\Delta\Omega_\mathrm{E}`,
+offset by :math:`180^\circ`. These correspond to rotating the screen by
+:math:`180^\circ` on the sky and this ambiguity in screen orientation cannot be
+resolved using single-telescope data. The angle :math:`\xi`, however, is
+restricted to the range :math:`0^\circ \leq \xi < 180^\circ` (because we use
+the convention that :math:`\xi` refers to the position angle of the *eastern*
+half of the line of lensed images). So, for the purpose of inferring
+:math:`\xi`, it is only necessary to consider one of the two
+:math:`\Delta\Omega_\mathrm{E}` solutions. We use Astropy's
+:py:class:`~astropy.coordinates.Angle` class and its
+:py:meth:`~astropy.coordinates.Angle.wrap_at` method to restrict the value of
+:math:`\xi` to its allowed range.
+
+.. jupyter-execute::
+
+    delta_omega_e = np.arctan(np.tan(chi_e) / np.cos(i_e))
+    xi = delta_omega_e + omega_e
+    xi = Angle(xi).wrap_at(180.*u.deg).deg * u.deg
+
+    print(f'xi: {xi.to(u.deg):8.2f}')
+
+
+The orientation of the pulsar's orbit
+-------------------------------------
+
+Knowing :math:`\xi`, it is possible to retrieve a relation between
+:math:`\Omega_\mathrm{p}` and :math:`i_\mathrm{p}` from the equation
+
+.. math::
+
+    \Omega_\mathrm{p} = \xi - \Delta\Omega_\mathrm{p},
+    \qquad \mathrm{with} \qquad
+    \tan( \Delta\Omega_\mathrm{p} ) = \frac{ \tan( \chi_\mathrm{p} ) }
+                                           { \cos( i_\mathrm{p} ) }.
+
+Again, for a given value of :math:`\chi_\mathrm{p}`, there are two possible
+solutions for :math:`\Delta\Omega_\mathrm{p}`, offset by :math:`180^\circ`.
+Hence, there are two possible :math:`i_\mathrm{p}`--:math:`\Omega_\mathrm{p}`
+relations, offset by :math:`180^\circ` in :math:`\Omega_\mathrm{p}`. We use
+Astropy's :py:class:`~astropy.coordinates.Angle` and
+:py:meth:`~astropy.coordinates.Angle.wrap_at` to restrict the values of
+:math:`\Omega_\mathrm{p}` to its allowed range of
+:math:`0^\circ \leq \Omega_\mathrm{p} < 360^\circ`.
+
+.. jupyter-execute::
+
+    i_p = np.linspace(0., 180., 181) << u.deg
+
+    delta_omega_p1 = np.arctan(np.tan(chi_p) / np.cos(i_p))
```

Comment:
I'd keep the 180 degree ambiguity inside the array as a second dimension, with
```
delta_omega_p = np.arctan(np.tan(chi_p) / np.cos(i_p)) + [[0], [180]] * u.deg
```

That way the next equations can all be single ones.

Reply:
Thanks, that's much better!